core/plugins/stack/terraform/mod.ts

```diff
@@ -0,0 +1,29 @@
+import { Introspector } from "../../../types.ts";
+import { introspect as introspectVersion } from "./version.ts";
+
+export default interface TerraformProject {
+  version?: string;
+}
+
+export const introspector: Introspector<TerraformProject | undefined> = {
+  detect: async (context) => {
+    return await context.files.includes("**/*.tf");
+  },
+  introspect: async (context) => {
+    const logger = await context.getLogger("terraform");
+    logger.info("Detected terraform file");
```

Comment:
Can you remove this info log? It presents redundant information:
![redundant](https://user-images.githubusercontent.com/1153190/147979321-e80b44b0-ce8a-4421-a096-f3d7ba4f9f73.png)

Reply:
I did the adjustments: 

-  in README: 
![image](https://user-images.githubusercontent.com/31106626/148061445-1df442a8-952e-4f50-b526-e75d77271534.png)

-  removed the info log: 
![image](https://user-images.githubusercontent.com/31106626/148061213-f29b1f4f-925b-48e5-9562-7b6ca5175817.png)